### PR TITLE
add ability to keep the saved video without moving it to the photo li…

### PR DESCRIPTION
**Pull request by ssp**
…brary

### CODE REVIEW

**Review by ManuelRosa — APPROVED** [review]

File: ios/Classes/ARCapture/ARCapture.swift [hunk 1]
```diff
@@ -127,14 +127,19 @@ open class ARCapture {
...
-    public func stop(_ complete: ((Bool)->())? = nil) {
+    public func stop(_ complete: ((Bool)->())? = nil, moveToPhotoLibrary: Bool = true) {
         self.status = .ready
         queue.sync { [weak self] in
             self?.lastPauseTime = nil
             self?.assetCreator?.stop { [weak self] in
                 if let url = self?.videoUrl {
                     if let complete = complete {
-                        self?.addVideoToLibrary(from: url, completed: complete)
+                        if moveToPhotoLibrary {
```
Comment (MaximilianKlein) [inline ios/Classes/ARCapture/ARCapture.swift:137]: moveToPhotoLibrary => addVideoToLibrary

should it be `moveToVideoLibrary` ?

Comment (ssp) [reply ios/Classes/ARCapture/ARCapture.swift:137]: I don't think there is a Video Library. It’s all in the library of the Photos app.

Comment (MaximilianKlein) [reply ios/Classes/ARCapture/ARCapture.swift:137]: but the naming is somehow inconsistent??

Comment (ssp) [reply ios/Classes/ARCapture/ARCapture.swift:137]: do we want to shave that yak now and clean up the code from the internet?

Comment (MaximilianKlein) [reply ios/Classes/ARCapture/ARCapture.swift:137]: No I wanted to understand why it said "addVideoToLibrary" I misunderstood. So when they named it wrongly.. It's fine for me.

**Review by MaximilianKlein — APPROVED** [review]
LGTM

**Pull request merged**